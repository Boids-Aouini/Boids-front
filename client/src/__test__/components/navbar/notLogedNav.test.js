import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NotLogedNav from '../../../components/navbar/notLogedNav';

describe('<NotLogedNav />', () => {
    describe('render()', () => {
        test('render without crashing', () => {
            shallow(<NotLogedNav />)
        })
        test('render component', () => {
            let wrapper = shallow(<NotLogedNav />)
            expect(toJson(wrapper)).toMatchSnapshot()
        })
    })

})
