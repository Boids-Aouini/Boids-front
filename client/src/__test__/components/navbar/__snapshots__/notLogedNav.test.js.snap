// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotLogedNav /> render() render component 1`] = `
<nav
  id="not-loged-nav"
>
  <Link
    class="notLogedNav-links"
    to="/"
  >
    Home
  </Link>
  <Link
    class="notLogedNav-links"
    to="/whyBoids"
  >
    Why Boids ?
  </Link>
  <Link
    class="notLogedNav-links"
    id="login"
    to="/login"
  >
    Login
  </Link>
</nav>
`;
